AlfrescoContent/AlfrescoContent/Classes/Swaggers/Models/SearchRequest.swift

```diff
@@ -44,18 +44,25 @@ public struct SearchRequest: Codable {
         self.defaults = defaults
         self.localization = localization
         self.filterQueries = filterQueries
-        self.facetQueries = facetQueries
-        self.facetFields = facetFields
-        self.facetIntervals = facetIntervals
         self.pivots = pivots
         self.stats = stats
         self.spellcheck = spellcheck
         self.scope = scope
         self.limits = limits
         self.highlight = highlight
         self.ranges = ranges
+        
+        self.facetQueries = facetQueries?.queries
+        if let facetFields = facetFields {
```

Comment:
reduce the line of code as below
self.facetIntervals = nil
if let facetIntervals = facetIntervals {
            self.facetIntervals = ["intervals": facetIntervals.intervals]
  }